### DISCUSSION TITLE
Add template hook to element index so that we can add new filters.

### DISCUSSION THREAD

**Discussion by johnrich85** [craftcms/cms#8646] [Ideas]
### Description
I'm looking to add some new filters (select fields within the toolbar) to the UI on the element index page and I'm not sure about the best way to do that currently. It's possible that it's only going to be achievable with JS.

I was thinking that if a template hook was added to the following template, it might make this less problematic.

/cms/src/templates/_elements/indexcontainer.html

Any thoughts/advice appreciated.

### Additional info

- Craft version: 3

**Comment by brandonkelly** [comment]
This will be possible when we add conditional filtering for elements (e.g. #1630).
